packages/worker/src/worker.ts

```diff
@@ -413,6 +417,7 @@ export class Worker<T extends WorkerSpec = DefaultWorkerSpec> {
   protected readonly numInFlightActivationsSubject = new BehaviorSubject<number>(0);
   protected readonly numInFlightActivitiesSubject = new BehaviorSubject<number>(0);
   protected readonly numRunningWorkflowInstancesSubject = new BehaviorSubject<number>(0);
+  private readonly runIdsToSpanContext = new Map<string, SpanContext>();
```

Comment:
@bergundy Let me know if there's a better place to keep this state

Reply:
This is fine.